articles/quickstart/webapp/nodejs/01-login.md

```diff
@@ -189,14 +197,30 @@ router.get('/callback', function (req, res, next) {
 // Perform session logout and redirect to homepage
 router.get('/logout', (req, res) => {
   req.logout();
+
+  var returnTo = util.format(
+    '%s://%s:%s',
+    req.protocol,
+    req.hostname,
+    req.connection.localPort
+  );
+  var logoutURL = new URL(
+    util.format('https://%s/logout', process.env.AUTH0_DOMAIN)
+  );
+  var queryString = querystring.stringify({
+    client_id: process.env.AUTH0_CLIENT_ID,
+    returnTo: returnTo
+  });
+  logoutURL.search = queryString;
+
   res.redirect('/');
 });
 
 module.exports = router;
 ```
 
 :::note
-This tutorial implements logout by closing the local user session. After logging out, the user's session in the Auth0 authentication server is still open. For other implementations, please refer to the [logout documentation](/logout). 
+This tutorial implements logout by closing the local user session and the user's session in the Auth0 authentication server as well. For other implementations, please refer to the [logout documentation](/logout).
```

Comment:
> ... and the user's session in the Auth0 authentication server

Consider:

> and the user's Auth0 session

Reply:
Also: 

https://auth0.com/docs/logout/

I think that's what this relative link is translated to ... if so, that's a 404

Reply:
There's an issue with trailing slashes. https://auth0.com/docs/logout works.